{"uuid":"8ff831d4-4c56-4091-b3a9-0bf39efc0402","historyId":"186ce1128e3043b3ecda186dfc3aa55b","fullName":"com.sqs.prestatests.PrestaCreateAccountTest.emailErrorTest","labels":[{"name":"package","value":"com.sqs.prestatests.PrestaCreateAccountTest"},{"name":"testClass","value":"com.sqs.prestatests.PrestaCreateAccountTest"},{"name":"testMethod","value":"emailErrorTest"},{"name":"parentSuite","value":"SQSTestSuite"},{"name":"suite","value":"PrestaTest"},{"name":"subSuite","value":"com.sqs.prestatests.PrestaCreateAccountTest"},{"name":"host","value":"Andys-MacBook-Pro-3.local"},{"name":"thread","value":"73414@Andys-MacBook-Pro-3.local.main(1)"}],"links":[{"name":"3860","type":"tms"}],"name":"Using an existing email in the create customer account form. Server side Error message should appear saying the email already exists.","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.NullPointerException","trace":"java.lang.NullPointerException\n\tat com.sqs.prestatests.PrestaCreateAccountTest.emailErrorTest(PrestaCreateAccountTest.java:154)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:497)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:661)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:744)\n\tat org.testng.TestRunner.run(TestRunner.java:602)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:380)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1226)\n\tat org.testng.TestNG.runSuites(TestNG.java:1144)\n\tat org.testng.TestNG.run(TestNG.java:1115)\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\n"},"stage":"finished","description":"Using an existing email in the create customer account form. Server side Error message should appear saying the email already exists.","start":1565952690907,"stop":1565952692128,"parameters":[{"name":"webDriverVersion","value":"74.0.3729.6"},{"name":"browser","value":"Chrome"}]}